Sources/App/Features/Ticket Hub/Controllers/TicketHubRouteController.swift

```diff
@@ -0,0 +1,310 @@
+import Fluent
+import Vapor
+
+struct TicketHubRouteController: RouteCollection {
+    func boot(routes: RoutesBuilder) throws {
+        routes.grouped(ValidTicketMiddleware()).group("ticket") { builder in
+            builder.get { req -> View in
+                guard let currentEvent = try await Event.query(on: req.db).filter(\.$name == "SwiftLeeds 2023").first() else {
+                // guard let currentEvent = req.storage.get(CurrentEventKey.self) else {
```

Comment:
🚨  This must be reverted before pushing to production. I've left it in so we can test on the branch deployment (and see what it will look like)